src/python/sympy_boilerplate.yaml

```diff
@@ -1075,24 +1091,28 @@ title: |
 code: |
   expr = (x + 2)*(x - 3)
   expr = expand(expr)
+  expr
```

Comment:
hmm, I wonder, if all of these (this one, and similar changes below) should just not assign to the `expr`, i.e.

```
expr = (x + 2)*(x - 3)
expand(expr)
```

this will show the expanded expression, because it's the last object on the stack in the last line

Reply:
Thank you for the suggestion @haraldschilly - I have updated all the snippets.